SS_Activation_OpenMC.py

```diff
@@ -0,0 +1,166 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Feb  8 08:16:12 2024
+
+@author: Anupama Rajendra
+"""
+import openmc
+import openmc.deplete
+import numpy as np
+from fractions import Fraction
```

Comment:
I was using this because of the 4/3 factor that appears in the spherical volume of depletable material but switched to 1.33 at some point. I now see that the Fraction module isn't necessary even when 4/3 is used -- getting rid of it